docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,385 @@
+   "source": [
+    "quantity_series = order_products_df.ww['quantity']\n",
+    "column_schema = quantity_series.ww.schema\n",
+    "column_schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f067db9a",
+   "metadata": {},
+   "source": [
+    "The `ColumnSchema` above has been pulled from the `'quantity'` column in the `order_products` table in the retail EntitySet. This is a **type definition**. \n",
+    "\n",
+    "If we look at the `order_products` table's Woodwork types below, we can see that there are several columns that will have similar `ColumnSchema` type definitions. If we wanted to describe subsets of those columns, we could define several `ColumnSchema` **type spaces**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bc2bfae6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es['order_products'].ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73257dcf",
+   "metadata": {},
+   "source": [
+    "Below are several `ColumnSchema`s that all would include our `quantity` column, but each of them describes a different type space. These `ColumnSchema`s get more restrictive as we go down:\n",
+    "\n",
+    "##### Entire Table\n",
+    "No restrictions have been placed; any column falls into this definition. This would include the whole DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6614c98",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork.column_schema import ColumnSchema\n",
+    "\n",
+    "ColumnSchema()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "299fc7d2",
+   "metadata": {},
+   "source": [
+    "An example of a Primitive with this `ColumnSchema` as its input type is the `IsNull` transform primitive.\n",
+    "\n",
+    "##### By Standard Tag\n",
+    "Only columns with the `numeric` tag apply. This can include Double, Integer, and Age logical type columns as well. It will not include the `index` column which, despite containing integers, has had its standard tags replaced by the `'index'` tag."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16c1a5a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ColumnSchema(semantic_tags={'numeric'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0932d05d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = es['order_products'].ww.select(include='numeric')\n",
+    "df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5ec95c8",
+   "metadata": {},
+   "source": [
+    "And example of a Primitive with this `ColumnSchema` as its input type is the `Mean` aggregation primitive.\n",
+    "\n",
+    "##### By Logical Type\n",
+    "Only columns with logical type of `Integer` are included in this definition. Does not require the `numeric` tag, so an index column (which has its standard tags removed) would still apply."
```

Comment:
@thehomebrewnerd I want to confirm that this is the actual behavior (that the columns returned by the select call below are the columns that this `ColumnSchema` would select) here and in the other sections. Based on my understanding of `match_by_schema` it will, but figured it was worth double checking.

Reply:
Yep, you are correct!